data/fields/sidewalk.json

```
@@ -0,0 +1,34 @@
+{
+    "key": "sidewalk:both",
+    "keys": [
+        "sidewalk:left",
+        "sidewalk:right"
+    ],
+    "reference": {
+        "key": "sidewalk"
+    },
+    "type": "directionalCombo",
+    "label": "Sidewalks",
+    "placeholder": "Yes, No, Mapped using separate ways",
```

```suggestion
    "placeholder": "Yes, No, Mapped Separately",
```

In order to...

1. match the `title` (and capitalization) of the option in question
2. fit in the text box.